src/components/ReportActionItem/ReportPreview.js

```diff
@@ -0,0 +1,159 @@
+
+        /** Outstanding amount in cents of this transaction */
+        total: PropTypes.number,
+
+        /** Currency of outstanding amount of this transaction */
+        currency: PropTypes.string,
+
+        /** Does the iouReport have an outstanding IOU? */
+        hasOutstandingIOU: PropTypes.bool,
+    }),
+
+    /** Session info for the currently logged in user. */
+    session: PropTypes.shape({
+        /** Currently logged in user email */
+        email: PropTypes.string,
+    }),
+
+    /** Popover context menu anchor, used for showing context menu */
+    contextMenuAnchor: PropTypes.shape({current: PropTypes.elementType}),
+
+    /** Callback invoked when View Details is pressed */
+    onViewDetailsPressed: PropTypes.func,
+
+    /** Callback for updating context menu active state, used for showing context menu */
+    checkIfContextMenuActive: PropTypes.func,
+
+    /** Whether the IOU is hovered so we can modify its style */
+    isHovered: PropTypes.bool,
+
+    ...withLocalizePropTypes,
+};
+
+const defaultProps = {
+    contextMenuAnchor: null,
+    isHovered: false,
+    iouReport: {},
+    onViewDetailsPressed: () => {},
+    checkIfContextMenuActive: () => {},
+    session: {
+        email: null,
+    },
+};
+
+const ReportPreview = (props) => {
+    const reportAmount = CurrencyUtils.convertToDisplayString(props.iouReport.total, props.iouReport.currency);
+    const managerEmail = props.iouReport.managerEmail || '';
+    const managerName = ReportUtils.getDisplayNameForParticipant(managerEmail, true);
+    const isCurrentUserManager = managerEmail === lodashGet(props.session, 'email', null);
+    return (
+        <View style={[styles.chatItemMessage, styles.mt4]}>
+            {_.map(props.action.message, (index) => (
+                <Pressable
+                    key={`ReportPreview-${props.action.reportActionID}-${index}`}
+                    onPress={props.onViewDetailsPressed}
+                    onPressIn={() => DeviceCapabilities.canUseTouchScreen() && ControlSelection.block()}
+                    onPressOut={() => ControlSelection.unblock()}
+                    onLongPress={(event) => showContextMenuForReport(event, props.contextMenuAnchor, props.chatReportID, props.action, props.checkIfContextMenuActive)}
+                    style={[styles.flexRow, styles.justifyContentBetween]}
+                    focusable
+                >
+                    <View>
+                        {props.iouReport.hasOutstandingIOU ? (
+                            <Text style={[styles.chatItemMessage, styles.cursorPointer]}>{props.translate('iou.payerOwesAmount', {owner: managerName, amount: reportAmount})}</Text>
+                        ) : (
+                            <View style={[styles.flexRow]}>
+                                <Text style={[styles.chatItemMessage, styles.cursorPointer]}>{props.translate('iou.payerSettled', {owner: managerName, amount: reportAmount})}</Text>
```

Comment:
there is no `owner` to replace here https://github.com/Expensify/App/blob/6851675645dd37a08a9c34023ac7af3695631db1/src/languages/en.js#L329

```suggestion
                                <Text style={[styles.chatItemMessage, styles.cursorPointer]}>{props.translate('iou.payerSettled', {amount: reportAmount})}</Text>
```

Reply:
technically this one isn't a blocker, my previous comment is though https://github.com/Expensify/App/pull/18486#discussion_r1191762665 we render `undefined owes [amount]` b/c the key is wrong.